### PR TITLE
Update Github link

### PR DESCRIPTION
**Pull request by voukau**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -14,10 +14,10 @@ <h1>マヌエル.com</h1>
         <h4>Best drunk purchase of 2020</h4>
         <img src="https://media.tenor.com/images/9ce092207d888099f3544cf6f9812b36/tenor.gif" alt="wink wink" class="thumb-index">
         <hr>
-        <h4>GitHub > <a href="https://github.com/eetkino">eetkino</a></h4>
+        <h4>GitHub > <a href="https://github.com/kinowashere">eetkino</a></h4>
```
Comment (kinowashere) [inline public/index.html:17]: There's unexpected behavior here.

**Review by kinowashere — CHANGES_REQUESTED** [review]
**QA Failed**

**Description** - eetkino appears as the text instead of kinowashere

**Expected** - kinowashere appears as the link's text and directs you to kinowashere's GitHub

**Steps to Repoduce**

1. Open `public/index.html`

File: public/index.html (hunk 1 above)
Comment (voukau) [reply public/index.html:17]: Investigating.

File: public/index.html [hunk 2]
```diff
@@ -14,7 +14,7 @@ <h1>マヌエル.com</h1>
         <h4>Best drunk purchase of 2020</h4>
         <img src="https://media.tenor.com/images/9ce092207d888099f3544cf6f9812b36/tenor.gif" alt="wink wink" class="thumb-index">
         <hr>
-        <h4>GitHub > <a href="https://github.com/kinowashere">eetkino</a></h4>
+        <h4>GitHub > <a href="https://github.com/kinowashere">kinowashere</a></h4>
```
Comment (kinowashere) [inline public/index.html:17]: Yup looks good

**Review by kinowashere — APPROVED** [review]
**QA Passed**

Great job! Looks good.

**Pull request merged**